runtime/j9vm/jvm.c

```diff
@@ -452,6 +456,74 @@ static const J9SignalMapping signalMap[] = {
 	{NULL, J9_SIG_ERR}
 };
 
+static void
+captureCommandLine(void)
+{
+#define ENV_VAR_NAME "IBM_JAVA_COMMAND_LINE"
```

Comment:
I suggest we call this OPENJ9_JAVA_COMMAND_LINE. It means changing the places that use it, but using OPENJ9 is more appropriate.

Reply:
Changing the name introduces the potential that both are defined: DDR should continue to recognize `IBM_JAVA_COMMAND_LINE` that may be found in existing system dump files. I will adjust things so `OPENJ9_JAVA_COMMAND_LINE` takes precedence.

Reply:
Fixed in 229e47cda1681285905983db3863a8c897ba9eab.